liquibase-integration-tests/src/test/groovy/liquibase/OracleTest.groovy

```diff
@@ -118,6 +119,25 @@ END;
         noExceptionThrown()
     }
 
+    def "verify Unique constraint is not created again when precondition fails because it already exists"() {
```

Comment:
Would it not be possible to run this test for _all_ supported databases? I'm not sure how much additional work it would take to do it  (are there not some tests that run for all supported database types?) and that way with (hopefully) not much effort, the test coverage of this precondition would be greatly increased.

Just a thought :) Thanks for your work here!